src-java/kilda-persistence-tinkerpop/src/main/java/org/openkilda/persistence/ferma/frames/converters/FlowApplicationConverter.java

```diff
@@ -28,14 +28,14 @@ public String toGraphProperty(FlowApplication value) {
         if (value == null) {
             return null;
         }
-        return value.name();
+        return value.name().toLowerCase();
```

Comment:
If it's the first release in the migration series, then how can we make sure that we can rollback this release? Any stored data in the lower case can't be read by the previous one. Am I right?